tests/TestSynthesis.cpp

```diff
@@ -340,15 +340,51 @@ TEST_CASE("Scheduler Testing", "[CScheduler]")
 		CHECK_ARRAY_CLOSE(ppfGroundBuffer, ppfSchedulerBuffer, iNumChannels, iLength, 1E-3);
 		
 	}
-
-	delete pLooper;
-	delete pOsc;
-	delete pSchedule;
 	for (int channel = 0; channel < iNumChannels; channel++)
 	{
 		delete[] ppfGroundBuffer[channel];
 		delete[] ppfSchedulerBuffer[channel];
 	}
 	delete[] ppfGroundBuffer;
 	delete[] ppfSchedulerBuffer;
+}
+
+// This test is just used as a lightweight and convenient way to track multi-threading -- no actual testing/comparisons are being made
+TEST_CASE("Multi-Threading Tests", "[MainProcessor]")
```

Comment:
Good idea. To clarify: since there's no `CHECK`s, what does a failure look like? An access violation/segfault? (And would that happen reliably with the old code, or unpredictably, as a race condition?)